src/jobs/publish_s3.yml

```diff
@@ -0,0 +1,35 @@
+description: Publish files to S3
+
+executor: aws
+
+parameters:
+  public:
+    type: boolean
+    description: Whether to store in the public bucket
+  path:
+    type: string
+    description: Path to store the files in S3
+  files:
+    type: string
+    description: Files to publish separated by space
+
+steps:
+  - attach_workspace:
+      at: ~/workdir
+
+  - run_aws:
+      cmd_name: Send files to S3
+      cmd: |
+        if [[ << parameters.public >> ]]; then
+          bucket=$AWS_PUBLIC_ARTIFACTS_BUCKET
+        else
+          bucket=$AWS_PRIVATE_ARTIFACTS_BUCKET
+        fi
+
+        for file in << parameters.files >> ; do
+          filename=$(basename $file)
+          AWS_PROFILE=production aws s3 cp $file $bucket/<< parameters.path >>/$(cat .version)/$filename
```

Comment:
This "AWS_PROFILE=production" seems a bit hardcoded here, no? Shouldn't we just have things available on the context when invoking the orb? just like the "AWS_PUBLIC_ARTIFACTS_BUCKET" shouldn't we have "AWS_ARTIFACTS_BUCKET_PROFILE" or something?

Also, can you explain me how this orb has access to the profiles? It is because it has access to the files from wherever place we call it? Is it because of the `at: ~/workdir`?

Reply:
> Also, can you explain me how this orb has access to the profiles? It is because it has access to the files from wherever place we call it?

It is because it uses the `run_aws` command that does all the aws configuration